docs/templating-guide/helper-functions.md

```diff
@@ -2,57 +2,57 @@
-| unix_time(optionalSeconds uint) float64                             | Returns the current Unix time (number of seconds elapsed since January 1, 1970 UTC) with the added optional seconds | `unix_time(10)` // 1639568278                                                                            |
-| url_decode(input string) string                                     | URL decodes the input string                                                                                        | `url_decode("https:%2F%2Fprojectdiscovery.io%3Ftest=1")` // https://projectdiscovery.io?test=1           |
-| url_encode(input string) string                                     | URL encodes the input string                                                                                        | `url_encode("https://projectdiscovery.io/test?a=1")` // https%3A%2F%2Fprojectdiscovery.io%2Ftest%3Fa%3D1 |
-| wait_for(seconds uint)                                              | Pauses the execution for the given amount of seconds                                                                | `wait_for(10)` // true                                                                                   |
+| Helper function                                                     | Description                                                                                                         | Example                                                                                 | Output                                                                                                                                                                                                                                                                                                                                                                                     |
+|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aes_gcm(key, plaintext interface{}) | AES GCM encrypts a string with key | `{{hex_encode(aes_gcm("AES256Key-32Characters1234567890", "exampleplaintext"))}}` |  `ec183a153b8e8ae7925beed74728534b57a60920c0b009eaa7608a34e06325804c096d7eebccddea3e5ed6c4` | 
```

Comment:
The table is not correctly formatted. 

e.g. in Intellij, there is a "Reformat Table" inspection, which does this for you:

![image](https://user-images.githubusercontent.com/13679401/175374666-1c619f80-ae11-4c77-aa10-1640dd0d4d9f.png)